### PR TITLE
Clean BlockVoting changes for WeylGovernance

### PR DESCRIPTION
**Pull request by john-osullivan**
Made a fresh head branch which didn't have any junk from rebase attempts, now it's a clean diff of just what I changed/added.  Again, pretty minimal:
- Made [a small change to `BlockVoting.sol`](https://github.com/Eximchain/quorum/compare/eximchain-integration...john-osullivan:weyl-pre-merge?expand=1#diff-b24082bc42bcd75a170e44d064d4f867R46)
- Recompiled the ABI & bytecode, put them in the appropriate places
- Regenerated the `core/quorum/binding.go` based on new ABI

Going to close the other PR now that this one is behaving.

### CODE REVIEW

**Review by Lsquared13 — APPROVED** [review]
LGTM. @eximchain137 should approve this one if possible and then I'll merge

**Comment by john-osullivan** [comment]
@eximchain137 Hey, could you take a quick look over this and hit approve so it can be merged?  Pretty sure you've already been building off these bindings for a little while.  Just want to merge this PR in so that the auditors have a complete copy of our geth.

**Review by eximchain137 — APPROVED** [review]
lgtm

**Pull request merged**
